Makefile

```diff
@@ -0,0 +1,4 @@
+js:
+	rm -rf var/cache/ ; npm run build-assets ; npm run copy-assets
```

Comment:
Deleting the cache is not strictly required for JS. Instead, it deletes the template and content cache. I'd put it in a separate `clear` task and create a `ui` task that depends on `js` and `clear`